ports/rp2/machine_uart.c

```diff
@@ -38,17 +38,58 @@
 #define DEFAULT_UART_BAUDRATE (115200)
 #define DEFAULT_UART_BITS (8)
 #define DEFAULT_UART_STOP (1)
+// UART 0 default pins
+#if defined(MICROPY_HW_UART0_TX)
+#define DEFAULT_UART0_TX MICROPY_HW_UART0_TX
```

Comment:
Please follow how it's done in #7513: just rename `DEFAULT_UART0_TX` to `MICROPY_HW_UART0_TX` in the initial objects below, and then this code here can be:
```
#if !defined(MICROPY_HW_UART0_TX)
#define MICROPY_HW_UART0_TX (0)
#define MICROPY_HW_UART0_RX (1)
#define MICROPY_HW_UART0_CTS (2)
#define MICROPY_HW_UART0_RTS (3)
#endif
```